juriscraper/pacer/docket_report.py

```diff
@@ -1106,19 +1151,21 @@ def _get_attachments(self, cells):
 
         result = []
         for row in rows:
-            result.append(
-                {
-                    "attachment_number": self._get_attachment_number(row),
-                    "description": self._get_description_from_tr(row),
-                    "page_count": self._get_page_count_from_tr(row),
-                    "file_size_str": self._get_file_size_str_from_tr(row),
-                    "pacer_doc_id": self._get_pacer_doc_id(row),
-                    # It may not be needed to reparse the seq_no
-                    # for each row, but we may as well. So far, it
-                    # has always been the same as the main document.
-                    "pacer_seq_no": self._get_pacer_seq_no_from_tr(row),
-                }
-            )
+            attachment = {
+                "attachment_number": self._get_attachment_number(row),
+                "description": self._get_description_from_tr(row),
+                "page_count": self._get_page_count_from_tr(row),
+                "file_size_str": self._get_file_size_str_from_tr(row),
+                "pacer_doc_id": self._get_pacer_doc_id(row),
+                # It may not be needed to reparse the seq_no
+                # for each row, but we may as well. So far, it
+                # has always been the same as the main document.
+                "pacer_seq_no": self._get_pacer_seq_no_from_tr(row),
+            }
+            file_size_bytes = self._get_file_size_bytes_from_tr(row)
+            if file_size_bytes is not None:
+                attachment["file_size_bytes"] = file_size_bytes
+            result.append(attachment)
```

Comment:
Same comment as above works here about setting the value to None and including the key.